### ISSUE TITLE
Chaos Dashboard: part of the chaos experiment event is missing

### ISSUE THREAD

**Issue by cwen0** [chaos-mesh/chaos-mesh#1989]
## Bug Report

**What version of Kubernetes are you using?**
  

Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-16T11:56:40Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.2", GitCommit:"59603c6e503c87169aea6106f57b9f242f64df89", GitTreeState:"clean", BuildDate:"2020-02-07T01:05:17Z", GoVersion:"go1.13.5", Compiler:"gc", Platform:"linux/amd64"} 

**What version of Chaos Mesh are you using?**
 

g1eny0ung/nirvana-ui branch  

**What did you do?**
 

1. Create a schedule object to schedule network chaos experiment (network loss)
2. Pause schedule task 
3. Restart the chaos experiment generated by schedule task 
4. Check the status of the chaos experiment 

**What did you expect to see?**. 

The chaos experiment was paused. 

**What did you see instead?** 

* Describe the chaos experiment: 

```
Status:
  Conditions:
    Reason:
    Status:  True
    Type:    Selected
    Reason:
    Status:  False
    Type:    AllInjected
    Reason:
    Status:  True
    Type:    AllRecovered
    Reason:
    Status:  True
    Type:    Paused
  Experiment:
    Container Records:
      Id:            default/web-show-87b9b8c96-s64cm
      Phase:         Not Injected
      Selector Key:  .
    Desired Phase:   Stop
  Instances:
    default/web-show-87b9b8c96-s64cm:  12
Events:
  Type    Reason           Age                   From          Message
  ----    ------           ----                  ----          -------
  Normal  FinalizerInited  8m1s                  finalizer     Finalizer has been inited
  Normal  Updated          8m1s                  finalizer     Successfully update finalizer of resource
  Normal  Started          7m3s (x3 over 8m1s)   desiredphase  Experiment has started
  Normal  Updated          7m3s (x6 over 8m1s)   desiredphase  Successfully update desiredPhase of resource
  Normal  Updated          7m3s (x9 over 8m1s)   common        Successfully update records of resource
  Normal  Applied          7m3s (x2 over 8m1s)   common        Successfully apply chaos for default/web-show-87b9b8c96-s64cm
  Normal  Paused           7m3s (x3 over 7m15s)  desiredphase  Experiment has been paused
  Normal  Recovered        7m3s (x2 over 7m15s)  common        Successfully recover chaos for default/web-show-87b9b8c96-s64cm
``` 

We found 8 events from the describe command, but I only found two events on Chaos Dashboard.
![7QqD68sIIV](https://user-images.githubusercontent.com/22956341/122360767-3ca75300-cf89-11eb-8e3e-296284d10169.png) 

In addition, Chaos Dashboard reported the chaos experiment has been restarted, but I check the chaos experiment, the status still was paused. 

**Output of chaosctl**

**Comment by YangKeao** [comment]
The paused `Schedule` will keep pausing the experiment created by it. I think it's the expected behavior.

And what's the connection between pausing and event?

**Comment by cwen0** [comment]
> The paused `Schedule` will keep pausing the experiment created by it. I think it's the expected behavior.
> 
> And what's the connection between pausing and event?

Yes, this is expected behavior, but the dashboard shouldn't report "启动成功".
